_partners/hyland.md

```diff
@@ -0,0 +1,5 @@
+---
+ name: Hyland
+ logo: '/assets/media/partners/hyland.png'
+ link: 'https://hyland.com'
+ ---
```

Comment:
@aborroy 
Please remove the extra single space at the beginning of lines 2-5 
(Jekyll isn't reading it correctly with that tiny issue) 
thanks